fuzzers/040-clk-hrow-config/Makefile

```diff
@@ -0,0 +1,22 @@
+N ?= 15
+
+include ../fuzzer.mk
+
+database: build/segbits_clk_hrow.db
+
+build/segbits_clk_hrow.rdb: $(SPECIMENS_OK)
+	${XRAY_SEGMATCH} -o build/segbits_clk_hrow.rdb $(addsuffix /segdata_clk_hrow_top_r.txt,$(SPECIMENS)) $(addsuffix /segdata_clk_hrow_bot_r.txt,$(SPECIMENS))
```

Comment:
Maybe define a variable for `$(addsuffix /segdata_clk_hrow_top_r.txt,$(SPECIMENS)) $(addsuffix /segdata_clk_hrow_bot_r.txt,$(SPECIMENS))` as it is used twice. If not wrap long lines. In general I think defining more variables helps makefile clarity.